cmd/config-host-add.go

```diff
@@ -57,9 +68,15 @@ EXAMPLES:
      providing the signature type.
      $ set +o history
      $ {{.HelpName}} mys3-iam https://s3.amazonaws.com \
-                 BKIKJAA5BMMU2RHO6IBB V8f1CwQqAcwo80UEIJEjc5gVQUSSx5ohQ9GSrr12 s3v4
+                 BKIKJAA5BMMU2RHO6IBB V8f1CwQqAcwo80UEIJEjc5gVQUSSx5ohQ9GSrr12 --api "s3v4"
      $ set -o history
 
+	4. Add S3 API compatible storage service under "mys3" alias, to use dns style bucket lookup. For security reasons turn off bash history momentarily.
+		 $ set +o history
+		 $ {{.HelpName}} mys3 https://s3.amazonaws.com \
```

Comment:
We can perhaps list a Minio example instead of S3 to show how users can use Minio with bucket dns style.

Reply:
yes, that's a good idea.done